### ISSUE TITLE
I wish Eleventy docs should have a clip to copy Code button

### ISSUE THREAD

**Issue by tarunsinghofficial** [11ty/eleventy#3412]
### Is your feature request related to a problem? Please describe.

I'm often frustrated when I want to quickly copy code snippets from the Eleventy documentation. Without a "Copy Code" button, I have to manually select and copy the text, which can be time-consuming and prone to errors.

### Describe the solution you'd like

I would like a "Copy Code" button added next to each code snippet in the Eleventy documentation. This button should allow users to quickly copy the code to their clipboard with a single click.

### Describe alternatives you've considered

One alternative is to continue manually selecting and copying the text, but this is less efficient and can lead to mistakes. Another option could be to use browser extensions that provide this functionality, but it's not as convenient as having it built directly into the documentation.

**Comment by tarunsinghofficial** [comment]
Let me know, If I should work on this! Please assign it to me. Thanks

**Comment by groenroos** [comment]
I agree - though maybe this should be an issue against https://github.com/11ty/11ty-website/ instead, since that's where the code of the docs site lives?
